packages/mux-player-react/src/lazy.tsx

```diff
@@ -52,42 +44,44 @@ const Fallback = (props: FallbackProps) => {
     However, by calling this placeholder "mux-player",
     it now gets the same CSS applied to it that the eventual "real" mux-player element will. 
     */
-    <mux-player
-      ref={intersectionRef}
-      data-mux-player-react-lazy-placeholder
-      className={className || ''}
-      style={{
-        // default mux-player styling
-        aspectRatio: '16/9',
-        display: 'block',
-        backgroundColor: 'var(--media-background-color, #000)',
-        width: '100%',
-        // user-implemented styles
-        ...style,
-        // and placeholder styles that are important enough to override the user's preferences
-        position: 'relative',
-        backgroundImage: placeholder ? `url(${placeholder})` : undefined,
-        backgroundRepeat: 'no-repeat',
-        backgroundSize: 'var(--media-object-fit, contain)',
-        backgroundPosition: 'var(--media-object-position, 50% 50%)',
-        '--controls': 'none',
-      }}
-      placeholder={placeholder}
-      // since there's a possibility that the bundle loads before Suspense clears this placeholder,
-      // we need to make sure that the placeholder isn't interactive and its player chrome in doesn't get rendered
-      nohotkeys
-      aria-hidden
-      tabIndex={-1}
-    >
-      <div
-        data-mux-player-react-lazy-placeholder-overlay
-        style={{
-          position: 'absolute',
-          inset: 0,
-          backgroundColor: 'var(--controls-backdrop-color, rgba(0, 0, 0, 0.6))',
-        }}
-      />
-    </mux-player>
+    <>
+      <mux-player
+        ref={intersectionRef}
+        data-mux-player-react-lazy-placeholder
+        placeholder={placeholder}
+        style={style}
+        className={className || ''}
+        // since there's a possibility that the bundle loads before Suspense clears this placeholder,
+        // we need to make sure that the placeholder isn't interactive and its player chrome in doesn't get rendered
+        nohotkeys
+        aria-hidden
+        tabIndex={-1}
+      >
+        <div data-mux-player-react-lazy-placeholder-overlay />
+      </mux-player>
+      <style>{
+        /* css */ `
+        [data-mux-player-react-lazy-placeholder]:not([bump-specificity]) {
```

Comment:
`mux-player:not([audio])` was taking precedence over this rule in that brief moment of "the bundle is loaded but suspense hasn't unloaded the placeholder". `:not([bump-specificty])` is my way of matching that specificity so that these styles hold on

Reply:
what about `mux-player[data-mux-player-react-lazy-placeholder]`? Should be same specificity without using unused attributes.

Reply:
does @gkatsev solution work @decepulis? that seems a bit more elegant code wise

Reply:
Great timing. I _just_ finished testing the solution. It works and it's pushed!